recipes/sregistry/meta.yaml

```
@@ -0,0 +1,81 @@
+{% set name = "sregistry" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3092be0059ae837be1cc69e4943610e729eb3facf49f1fed05a37a75349adb48
+
+build:
+  number: 0
+  entry_points:
+    - sregistry=sregistry.client:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [win]
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - setuptools
+  run:
+    - awscli >=1.16.19
+    - boto3 >=1.7.83
+    - dropbox >=8.5.1
+    - globus-sdk >=1.5.0
+    - google-api-python-client >=1.6.4
+    - google-cloud-storage >=1.4.0
+    - oauth2client >=3.0
+    - pygments >=2.1.3
+    - python >=3
+    - requests >=2.18.4
+    - requests-toolbelt >=0.8.0
+    - retrying >=1.3.3
+    - spython
+    - sqlalchemy
+
+test:
+  imports:
+    - sregistry
+    - sregistry.auth
+    - sregistry.client
+    - sregistry.database
+    - sregistry.logger
+    - sregistry.main
+    - sregistry.main.aws
+    - sregistry.main.base
+    - sregistry.main.docker
+    - sregistry.main.dropbox
+    - sregistry.main.gitlab
+    - sregistry.main.globus
+    - sregistry.main.google_build
+    - sregistry.main.google_drive
+    - sregistry.main.google_storage
+    - sregistry.main.hub
+    - sregistry.main.nvidia
+    - sregistry.main.registry
+    - sregistry.main.s3
+    - sregistry.main.templates
+    - sregistry.main.templates.build
+    - sregistry.main.workers
+    - sregistry.tests
+    - sregistry.utils
+  commands:
+    - sregistry --help
+
+about:
+  home: https://github.com/singularityhub/sregistry-cli
+  license: Mozilla Public 2.0 (MPL 2.0)
```

[SPDX abbreviations](https://spdx.org/licenses/) are generally preferred:
```suggestion
  license: MPL-2.0
```